### DISCUSSION TITLE
Laragon on docker or CLI

### DISCUSSION THREAD

**Discussion by aaditya-cpu** [leokhoa/laragon#381] [General]
Hey, is there a DOcker-based or LInux-based deb file that I use to manage laragon via CLI? It has been quite some time I am using laragon. I was wondering if There was a LINux-based utility that could help me run it on my server

**Comment by Pen-y-Fan** [comment]
No, Laragon only works on Windows. 

If you have experience with Wine, it may be worth trying. Laragon requires MS runtime c++ libraries, which may be a problem.
